addons/account/models/account_journal_dashboard.py

```diff
@@ -611,6 +611,9 @@ def _get_open_payments_query(self):
             ('move_id.payment_state', 'in', ('not_paid', 'partial')),
             ('date_maturity', '!=', False),
             ('amount_residual', '<', 0),
+            ('account_id.reconcile', '=', True),
```

Comment:
Payable/receivable accounts will be reconcilable anyway, so you don't need this

(below as well)